# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1325169498 28800
#      Thu Dec 29 06:38:18 2011 -0800
# Branch stable
# Node ID 0a051c4062429de4807c308fa376fbb5e4bdc4e4
# Parent  94e2a76f1e5a09c4f58720a8747f636ed238d2e3
doc: Fix incorrect xref usage in warning_ids.m

warning_ids.m: Fix incorrect xref usage.

diff --git a/scripts/miscellaneous/warning_ids.m b/scripts/miscellaneous/warning_ids.m
--- a/scripts/miscellaneous/warning_ids.m
+++ b/scripts/miscellaneous/warning_ids.m
@@ -118,18 +118,18 @@
 ## Octave will warn when parsing autoload() function calls with relative
 ## paths to function files.  This usually happens when using autoload()
 ## calls in PKG_ADD files, when the PKG_ADD file is not in the same
 ## directory as the .oct file referred to by the autoload() command.
 ## By default, the @code{Octave:autoload-relative-file-name} warning is enabled.
 ##
 ## @item Octave:broadcast
 ## Warn when performing broadcasting operations.  By default, this is
-## enabled.  @xref{Broadcasting} in the chapter Vectorization and Faster Code
-## Execution of the manual.
+## enabled.  See @ref{Broadcasting} in the chapter Vectorization and Faster
+## Code Execution of the manual.
 ##
 ## @item Octave:built-in-variable-assignment
 ## By default, the @code{Octave:built-in-variable-assignment} warning is
 ## enabled.
 ##
 ## @item Octave:divide-by-zero
 ## If the @code{Octave:divide-by-zero} warning is enabled, a
 ## warning is issued when Octave encounters a division by zero.
